app/scrapers/generalized.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+import os, json, sys
+import requests
+from bs4 import BeautifulSoup
+
+defaultHeaders = {
```

Comment:
Can we make HEADERS a class constant so that all Scraper instances can share a single constant?

Reply:
:+1: